Documentation/getting-started-meson-build.md

```diff
@@ -20,6 +20,10 @@ The following dependencies are needed to compile/build/run.
 * [SSAT](https://github.com/myungjoo/SSAT) (optional. required for unit testing)
 * gtest (optional. required for unit testing)
 
+The minimal requirement to build nnstreamer with default configuration is
+```bash
+$ sudo apt install meson gcc g++ libglib2.0-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev
```

Comment:
I think ```ninja-build``` should be one of them at least.

Reply:
`meson` depends on `ninja-build`. So, it is automatically installed.
Should I add it explicitly?

Reply:
@kparichay Official debs do, but ours do not. 

Alternatively, we can add depends-on-ninja-build to our meson package.

Reply:
aah, then for now, I have added `ninja-build` here.
I will add as depends-on to our meson package. 
Once, it is added, I will remove from here later.
Thanks for pointing this out.